### PR TITLE
Can submit DD1299 (#154655011)

### PR DESCRIPTION
**Pull request by tinyels**

### CODE REVIEW

File: src/scenes/DD1299/ducks.js [hunk 1]
```diff
@@ -32,19 +53,49 @@ export function loadSchema() {
...
       return Object.assign({}, state, {
         schema: action.schema.definitions.CreateForm1299Payload,
-        hasError: false,
+        hasSchemaError: false,
       });
     case LOAD_SCHEMA_FAILURE:
-      return Object.assign({}, state, { schema: {}, hasError: true });
+      return Object.assign({}, state, { schema: {}, hasSchemaError: true });
+    case SUBMIT_SUCCESS:
+      return Object.assign({}, state, { hasSubmitSuccess: true });
```
Comment (kilbergr) [inline src/scenes/DD1299/ducks.js]: should this have `hasSubmitError: false` ?

File: src/scenes/DD1299/ducks.js [hunk 2]
```diff
@@ -21,6 +26,22 @@ export const createLoadSchemaFailure = error => ({
   error,
 });
 
+export const createRequestSubmit = () => ({
```
[the remark this answers is not in the document]
Comment (tinyels) [reply src/scenes/DD1299/ducks.js]: these are for loading the schema, the submit ones are for submitting the form.  Could be split into separate files/reducers but I think the schema stuff is going to get moved eventually anyway.

File: src/scenes/DD1299/ducks.js (hunk 1 above)
Comment (tinyels) [reply src/scenes/DD1299/ducks.js]: Good catch!

File: src/scenes/DD1299/ducks.js [hunk 3]
```diff
@@ -21,6 +26,22 @@ export const createLoadSchemaFailure = error => ({
...
+
+export const createSubmitFailure = error => ({
+  type: SUBMIT_FAILURE,
+  error,
+});
+
+export const createSubmitReset = () => ({
+  type: SUBMIT_RESET,
+});
 // Action Creator
```
[the remark this answers is not in the document]
Comment (tinyels) [reply src/scenes/DD1299/ducks.js]: Fixing now

**Pull request merged**